client/landing/jetpack-cloud/controller.js

```diff
@@ -33,7 +34,28 @@ export function jetpackCloud( context, next ) {
 }
 
 export function jetpackBackups( context, next ) {
-	context.primary = <div>This is the Jetpack Backup landing page!</div>;
+	context.primary = (
```

Comment:
I think we should add this example component to a storybook instead of here.

Reply:
I think a storybook/devdocs example wouldn't hurt, but this is also useful to have as we iterate on the Jetpack Cloud.

Reply:
So, I tested and it looks like devdocs doesn't function with JP Cloud. Made a ticket to rectify: 1156570014567299-as-1159444362918521

Regarding storybook, I'm confused- that only operates on `@automattic/packages`, not components in the `client` dir?

Reply:
Resolved with 5d87e8a125d0186d034ccbed75be6155b2431ae1